speakeasy-api/speakeasy
# feat: add LLM suggestions to CLI

A sample CLI that communicates with downstream speakeasy API and LLM server.

Normal Testing (CLI)
1. Make sure you have OPENAI_API_KEY set.
2. go run main.go validate openapi --schema openapi-invalid.yaml --fix

Testing with Local API
Local Testing

cd speakeasy-registry (API)
1 Run local registry API
2. Make docker-llm (wait until Chroma is up to continue, you should see Uvicorn running on http://0.0.0.0:8000)
3. export SPEAKEASY_SERVER_URL=http://localhost:35290
